packages/pds-ember/package.json

```diff
@@ -2,21 +2,89 @@
   "name": "@hashicorp/pds-ember",
   "version": "0.0.10",
   "description": "Ember addon for building Structure-styled UIs",
-  "main": "index.js",
-  "scripts": {
-    "test": "echo \"Error: no test specified\" && exit 1"
-  },
-  "repository": {
-    "type": "git",
-    "url": "git+https://github.com/hashicorp/structure.git"
-  },
   "keywords": [
     "ember-addon"
   ],
-  "author": "Ryan Johnson",
   "license": "MPL-2.0",
+  "author": "Ryan Johnson",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hashicorp/structure.git"
+  },
   "bugs": {
     "url": "https://github.com/hashicorp/structure/issues"
   },
-  "homepage": "https://github.com/hashicorp/structure#readme"
+  "homepage": "https://github.com/hashicorp/structure#readme",
+  "directories": {
+    "doc": "doc",
+    "test": "tests"
+  },
+  "scripts": {
+    "build": "ember build --environment=production",
+    "lint:hbs": "ember-template-lint .",
+    "lint:js": "eslint .",
+    "start": "ember serve",
+    "test": "npm-run-all lint:* test:*",
+    "test:ember": "ember test",
+    "test:ember-compatibility": "ember try:each",
+    "lint-all": "yarn lint:hbs && yarn lint:js",
+    "g:component": "ember g pds-component",
+    "d:component": "ember d pds-component",
+    "storybook": "start-storybook -p 6006 -s dist",
+    "storybook:build": "build-storybook"
+  },
+  "dependencies": {
+    "broccoli-funnel": "^3.0.3",
+    "broccoli-merge-trees": "^4.2.0",
+    "ember-cli-babel": "^7.22.1",
+    "ember-cli-htmlbars": "^5.3.1",
+    "ember-cli-sass": "^10.0.1",
+    "ember-cli-string-utils": "^1.1.0",
+    "ember-svg-jar": "^2.2.3",
+    "sass": "^1.26.11"
```

Comment:
How many of these should truly be a dependency? Can any/all of these be moved into `devDependencies`?

Reply:
According to https://cli.emberjs.com/release/writing-addons/addon-dependencies/, there are cases where we need `dependencies` for the addon itself.  However, we'll re-evaluate this list in a future PR.